packages/@ama-sdk/core/src/fwk/clients/api-fetch-client.ts

```diff
@@ -152,11 +152,25 @@ export class ApiFetchClient implements ApiClient {
     } catch (e: any) {
       exception = new ResponseJSONParseError(e.message || 'Fail to parse response body', response && response.status || 0, body, {apiName, operationId, url, origin});
     }
+    let statusCodeReviver: ReviverType<T> | undefined = typeof reviver === 'function' || typeof reviver === 'undefined' ?
+      reviver : typeof response?.status !== 'undefined' && reviver[response.status] || undefined;
+    const isSuccessResponseCode = !!response?.status && response.status >= 200 && response.status < 300 && response.status !== 204;
+    if (isSuccessResponseCode && typeof reviver === 'object' && !Object.prototype.hasOwnProperty.call(reviver, response!.status)) {
+      const fallback = Object.entries(reviver)?.reduce((acc: {statusCode: number; reviver: ReviverType<T> | undefined}, [key, value]) => {
+        if (+key < acc.statusCode && +key !== 204) {
+          return {statusCode: +key, reviver: value};
+        }
+        return acc;
+      }, {statusCode: Number.MAX_SAFE_INTEGER, reviver: undefined});
+      statusCodeReviver = fallback?.reviver;
+      // eslint-disable-next-line no-console
+      console.error(`Unknown status code ${response?.status || 'undefined'} for ${operationId || 'unknown endpoint'} - fallback to status code ${fallback?.statusCode}`);
```

Comment:
It's not more a 'warning' or 'info' than an 'error' ? 
Same as other logs related to APIs missing fields :)